src/index.html

```diff
@@ -19,9 +19,423 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/main.scss"
```

Comment:
Same issue here, you are linking to a .scss file 'styles/main.scss'. Please correct this.